### PR TITLE
Some fixes and adjustments, added a new example

### PR DESCRIPTION
**Pull request by XavierCLL**

### CODE REVIEW

**Comment by XavierCLL** [comment]
hello Gena! new changes come in the following days...

File: examples/test-hello-image.py [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-
-image = ee.Image.pixelLonLat() \
-    .add([180, 90]).divide([360, 180])
-
-# image = image.multiply(50).sin()
-
-Map.addLayer(image, {}, 'coords', True)
-
-
-
```
Comment (gena) [inline examples/test-hello-image.py]: Why is this file deleted?

Comment (XavierCLL) [reply examples/test-hello-image.py]: I didn't delete, I renamed the example files and added one

Comment (gena) [reply examples/test-hello-image.py]: Ah, I did not notice this. I've merged everything into master.
